docker-compose.yml

```diff
@@ -6,6 +6,12 @@ services:
       - '5432:5432'
     environment:
       POSTGRES_DB: accounts-js-tests-e2e
+  postgres_password:
```

Comment:
Would it be possible to reuse the e2e database image?

Reply:
yes absolutely, I realised the tests were running in band after I created the PR. So there shouldn't be any conflicts with the e2e tests.